src/llmq/quorums_signing_shares.cpp

```diff
@@ -1134,6 +1141,7 @@ void CSigSharesManager::WorkThreadMain()
             if (!workInterrupt.sleep_for(std::chrono::milliseconds(100))) {
                 return;
             }
+            lastSendTime = 0;
```

Comment:
Don't think this one is needed, even though in current setup it won't hurt. But if we would decide to use a larger timeout for "send" part than for "sleep" one then this would be completely wrong actually, so I'd prefer to drop this line.

Reply:
Removed that line